app/models/recipe.rb

```diff
@@ -16,6 +17,10 @@ class Recipe < ActiveRecord::Base
 
   acts_as_taggable
 
+  def forked_from
+    Recipe.where(id: forked_from_recipe_id).first
```

Comment:
and I was off track aswell... Of course `Recipe.find(forked_from_recipe_id)` is the way to go :)